### ISSUE TITLE
Jupyter Lab debugger

### ISSUE THREAD

**Issue by fixrie** [jupyter-xeus/xeus-r#86]
When using XEUS-R in combinaison with Jupyter Lab, it seems that the onboarded debugger is not supported by XEUS-R kernel.

**Comment by anutosh491** [comment]
Yes currently only the xeus-python and the xeus-robot kernels have debugger support enabled out of all xeus kernels 
https://jupyterlab.readthedocs.io/en/latest/user/debugger.html#requirements
